styles/tokens/input.json

```diff
@@ -768,7 +768,7 @@
       },
       "z-index": {
         "comment": "The z-index applied to the icon inside Inputs.",
-        "value": "{layer.height.xs.value}",
+        "value": "0",
```

Comment:
This is specifically for icons inside of inputs. When a tooltip is near, it's on the same level, which interferes if the tooltip is near the icon -- specifically in Percentage.

This change is fine because it's not affecting the render at all, and it's still a changeable token if the project deems necessary.